package.json

```diff
@@ -46,6 +48,7 @@
     "@nrwl/jest": "9.4.5",
     "@nrwl/nx-plugin": "^9.4.5",
     "@nrwl/workspace": "9.4.5",
+    "@types/fs-extra": "^9.0.1",
```

Comment:
Is this needed? I get type support without it.

Reply:
Yes. https://github.com/jprichardson/node-fs-extra#typescript

We're not using `fs-extra` in TypeScript at this time, but we may in the future. We should make it a habit to install `@types` if they exist and types aren't shipped with the dependency. That's the TypeScript game we play.